src/main/java/org/prebid/server/auction/ImplicitParametersExtractor.java

```diff
@@ -101,4 +101,9 @@ public Integer secureFrom(HttpRequestContext httpRequest) {
                 || StringUtils.equalsIgnoreCase(httpRequest.getScheme(), "https")
                 ? 1 : null;
     }
+
+    public Integer gpcFrom(HttpRequestContext httpRequest) {
+        final String gpcAsString = httpRequest.getHeaders().get(HttpUtil.SEC_GPC);
+        return "1".equals(gpcAsString) ? Integer.parseInt(gpcAsString) : null;
```

Comment:
Why do you try to parse "1" to int if you can just return it

Reply:
:)